src/common/styles.js

```diff
@@ -0,0 +1,29 @@
+function applyStyleMethod (func) {
+    var args = Array.prototype.slice.call(arguments, 1);
+    return "\"" + func + "(" + args.map(function (arg) {
+        return arg.replace(/"/g, '');
+    }).join(', ') + ")\"";
+};
+
+
+function createStyleFunc (func) {
+    return function () {
+        var args = Array.prototype.slice.call(arguments, 0);
+        args.unshift(func);
+        return applyStyleMethod.apply(null, args);
+    };
+}
+
```

Comment:
the es5 here makes it look like something complex is happening. maybe it cannot be es6 for some reason?

if it can, then it's this:

```js
const applyStyleMethod = (func, ...args) => {
  const unquotedArgs = args.map(arg => arg.replace(/"/g, '')).join(', ')
  return `"${func}(${unquotedArgs})"`
};

const createStyleFunc = func => (...args) => applyStyleMethod(func, ...args)
```

Reply:
That's true.
I'll see if it works 👍

Reply:
I think stylable doesn’t transpire es6, but please check

Reply:
> I think stylable doesn’t transpire es6, but please check

Yeah, that's what I thought. But i'll try.

Reply:
it's meh, i just stumbled upon and couldn't resist :) you don't have to do this

Reply:
Done!